message_diffs/_layout.email.html.erb.diff

```diff
@@ -1,13 +1,13 @@
---- ../../../app/messages/_layout.email.html.erb	2020-10-27 05:40:36.661361297 +0000
-+++ messages/_layout.email.html.erb	2020-11-02 23:47:08.166654343 +0000
+--- ../../../app/messages/_layout.email.html.erb	2020-12-16 22:03:48.792768097 +0000
++++ messages/_layout.email.html.erb	2021-02-12 21:18:24.739103447 +0000
```

Comment:
This patch's heading is weird: the before-file is ../../../app/messages/_layout.email.html.erb, and the after-file is messages/_layout.email.html.erb.  Can you explain why the paths are different?

Reply:
the top line is referencing the file in Canvas. The reason I've added the diffs is in case the templates are updated in Canvas, we may be able to automatically apply these diffs to the new version, or at the very least, see what the intended changes are and make the again manually.

Reply:
Okay, now I see the correspondence between the diffs and the full files in the review.